admin_guide/manage_nodes.adoc

```diff
@@ -595,10 +597,7 @@ volumeConfig:
   localQuota:
     perFSGroup: null    <8>
 volumeDirectory: /var/lib/origin/openshift.local.volumes
-
-Events:  <none>
```

Comment:
@mhutter Why remove the `Events:  <none>`? Is that not in the ConfigMap?

Reply:
No, it is not. It's part of the output of an `oc describe ...` command.